src/js/components/award/financialAssistance/CFDATree.jsx

```diff
@@ -106,7 +109,8 @@ export default class CFDATree extends React.Component {
             cells.push(cell);
         });
         this.setState({
-            virtualChart: cells
+            virtualChart: cells,
+            removedCFDAs: removedCFDAs.length
```

Comment:
`removedCFDAs`  was an array, now it's a string?

```suggestion
isPartialTree: removedCFDAs.length > 0
```